tests/test/transport/test_message_endpoint_client.cpp

```diff
@@ -224,6 +225,83 @@ TEST_CASE_METHOD(SchedulerTestFixture,
     }
 }
 
-#endif
+TEST_CASE_METHOD(SchedulerTestFixture, "Test direct messaging", "[transport]")
+{
+    std::string expected = "Direct hello";
+    const uint8_t* msg = BYTES_CONST(expected.c_str());
+
+    std::string inprocLabel = "direct-test";
+
+    AsyncDirectSendEndpoint sender(inprocLabel, TEST_PORT);
```

Comment:
Afaict `TEST_PORT` here is passed as the `timeoutMs` argument.

Reply:
Yes, good spot.